scss/product/components/_dropdown.scss

```diff
@@ -9,13 +9,19 @@
   @include caret;
 }
 
+//three dot menu toggle for drop down
```

Comment:
where we use this?

Reply:
We are not using this. I have removed the unused css